src/ActivityRepository.js

```diff
@@ -103,25 +103,16 @@ class ActivityRepository {
     return this.user.slice(index - 6, index + 1);
   }
 
-  getPositiveStepTrends() {
+  getStepTrends(positive) {
     return this.user.reduce((acc, day, index) => {
       if (index < 2) {
         return acc;
       }
-      if ((day.numSteps > this.user[index - 1].numSteps) &&
+      if (positive && (day.numSteps > this.user[index - 1].numSteps) &&
```

Comment:
I like that you bundled it all together, I don't know if it's just me but this is still a little hard to read and understand, I'm not used to using the index argument in reduce, do you think you can elaborate on that a little bit?

Reply:
Sure! That's the how the code was written when we received it, but I think I understand what's going on there. Since reduce effectively loops through each block of user activity data, that index represents the index of the current one being processes. It uses that current index, to compare either the two indices ahead of it  or behind it to assess if that index represents an increase in steps over the last two indices, or a decrease in the case of a negative trend.

Reply:
OH! That makes a lot more sense now, good on you to be able to take this and understand the fine intricacies of it.

Reply:
This looks good! My only potential change is that the 'positive' parameter might be a little confusing. Perhaps something like 'change' or 'direction'?  It is hard to come up with a name for that one.